### ISSUE TITLE
Backlog: Fix Redis intializer

### ISSUE THREAD

**Issue by murny** [ualbertalib/Hydranorth2#142]
Found in this PR: https://github.com/ualbertalib/Hydranorth2/pull/141

The redis setup we inherited from sufia's generator is pretty poorly done. `redis.yml` is probably not needed as these are already the default configurations (localhost and 6379) for the redis gem. If we don't force these configs when initialising redis, we will also be able to use the `REDIS_URL` ENV variable as well. Which is a far better way of doing this.

Seems to be a legacy setup from sufia that forced it onto the consuming app when they were still using resque? Is anything else using redis now that sufia moved to activejob? Maybe this ticket can be tied together with the work of when we configure ActiveJob (sidekiq or something). 

So to summarize: We should be able to remove redis.yml and rewrite redis initializer smarter when configuring our ActiveJob queue tool of choice.
